homeassistant/components/unifi_access/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow for the UniFi Access integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import uiaccessclient
+import urllib3.exceptions
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_API_TOKEN, CONF_HOST, CONF_VERIFY_SSL
+from homeassistant.core import HomeAssistant
+
+from .const import DEFAULT_HOST, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default=DEFAULT_HOST): str,
+        vol.Required(CONF_API_TOKEN): str,
+        vol.Required(CONF_VERIFY_SSL): bool,
+    }
+)
+
+
+class UniFiAccessConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for UniFi Access."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Process configuration form."""
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
+            info = await _validate_input(self.hass, user_input, errors)
+            if not errors:
+                return self.async_create_entry(title=info["title"], data=user_input)
+
+        return self.async_show_form(
+            step_id="user", data_schema=STEP_USER_DATA_SCHEMA, errors=errors
+        )
```

Comment:
So we should also prevent that the same service or device is able to be set up twice. So either make sure we get something unique from the service, or use `self._async_abort_entries_match()`

Reply:
I've added `self._async_abort_entries_match()` to abort for non-unique host.